src/tests/ftest/util/daos_utils.py

```diff
@@ -297,7 +297,28 @@ def pool_set_attr(self, pool, attr, value, sys_name=None):
 
         """
         return self._get_result(
-            ("pool", "set-attr"), pool=pool, attr=attr, value=value,
+            ("pool", "set-attr"), pool=pool, attr=':'.join([attr, value]),
+            sys_name=sys_name)
+
+    def pool_set_attrs(self, pool, attrs, sys_name=None):
+        """Set multiple pool attributes.
+
```

Comment:
Just FYI that once this PR lands, I'd like to personally reconcile the `set_attr` and `set_attrs` such that all tests just use your new version.